MobiusCore/Source/Mobius.swift

```diff
@@ -218,3 +195,26 @@ class LoggingUpdate<Model, Event, Effect> {
         return result
     }
 }
+
+private extension EffectHandler {
+    /// When using an `EffectHandler` to handle effects in a Mobius loop, at least one `EffectHandler` must respond to each event.
+    /// We cannot guarantee this in an `EffectHandler` or even in a composition of `EffectHandler`s, since we do not know how they will be nested.
+    /// We must therefore guarantee this property at the point where we connect our `EffectHandler` to the loop.
+    var asLoopConnectable: AnyConnectable<Effect, Event> {
+        return AnyConnectable { dispatch -> Connection<Effect> in
+            let effectHandlerConnection = self.connect(dispatch)
+            return Connection(
+                acceptClosure: { effect in
+                    if let executeEffect = self.canHandle(effect) {
+                        executeEffect(dispatch)
+                    } else {
+                        MobiusHooks.onError("More than one effect handler handling effect: \(effect)")
```

Comment:
Isn't this also possibly 'no effect handler handling effect'?

Reply:
Yes, and that is actually the only error it can be. `More than one effect handler` can only be caught in the composition of EffectHandlers, not when attaching to the loop.

Reply:
Fixed in the relevant commit.